packages/terra-tag/src/RollupTag.jsx

```diff
@@ -0,0 +1,99 @@
+import React, {
+  useRef,
+} from 'react';
+import {
+  KEY_SPACE, KEY_RETURN,
+} from 'keycode-js';
```

Comment:
Multiple lines are unnecessary for less than 3 items.
```suggestion
import React, { useRef } from 'react';
import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
```